.env.template

```diff
@@ -8,6 +8,11 @@ TRAEFIK_PASSWORD=$2y$05$klVQBa8nTDvU7euvEAuJROeAF7.vSPiX3ZWG7KcAowATXp7IJa.k.
 PROMETHEUS_USERNAME=dashboard
 PROMETHEUS_PASSWORD=$2y$05$klVQBa8nTDvU7euvEAuJROeAF7.vSPiX3ZWG7KcAowATXp7IJa.k.
 
+# basic http auth
+UPLOAD_USERNAME=
+UPLOAD_PASSWORD=
```

Comment:
default value suggestion:
upload
banana (needs to be copied from the encrypted version above)